docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -101,9 +101,19 @@ def get_ogcserver_accesscontrol_config(self):
 
     def get_ogcserver_accesscontrol(self):
         if self.single:
+            if "MAP" in parameters:
+                raise GMFException("The map parameter should not be provided")
             return self.ogcserver_accesscontrol
         else:
             parameters = self.serverInterface().requestHandler().parameterMap()
+            if "MAP" not in parameters:
+                raise GMFException("The map parameter should be provided")
```

Comment:
What does QGIS return in such case ?
status 500 ?

Reply:
Note that you could return an empty OGCServerAccessControl to be consistent (no access).

Reply:
It's possible but it is relay an error, We can't mix the tow comportment...